pandas/_libs/reduction.pyx

```diff
@@ -628,7 +628,7 @@ cdef class BlockSlider:
             arr.shape[1] = 0
 
 
-def reduce(arr, f, axis=0, dummy=None, labels=None):
+def do_reduce(arr, f, axis=0, dummy=None, labels=None):
```

Comment:
I don't particular like this name, we don't have anywhere we call things do_*, how about
compute_reduction

Reply:
sure

Reply:
(and its such an easy change to make because I can grep for it!)